src/roku-types/data.json

```diff
@@ -10285,23 +10350,29 @@
                     "params": [
                         {
                             "default": null,
+                            "description": "An associative array with the audio format to be checked. The general format of the associative arrays for CanDecodeAudio() is similar to the parameter and return associative arrays used in [CanDecodeVideo()](https://developer.roku.com/docs/references/brightscript/interfaces/ifdeviceinfo.mdcandecodevideovideo_format-as-object-as-object):  <table><thead><tr><th>Key</th><th>Type</th><th>Value</th><th>Requirement</th></tr></thead><tbody><tr><td>Codec</td><td>string</td><td>Specifies the audio codec: “aac”, “ac3”, “eac3”, \"alac\", \"flac\", “mp2”, “mp3”, “vorbis”, “wma”, “wma pro”, “dts”</td><td>Required</td></tr><tr><td>Profile</td><td>string</td><td>Specifies the codec profile:<table><thead><tr><th>Codec</th><th>Profile</th></tr></thead><tbody><tr><td>\"h.265/h.265\"</td><td>\"main\", \"high\", \"main 10\"</td></tr><tr><td>\"vp9\"</td><td>\"profile 0\", \"profile 2\"</td></tr><tr><td>\"AAC codec profiles\"</td><td>\"mp2 lc\", \"mp4 he\"</td></tr></tbody></table></td><td>Optional</td></tr><tr><td>Level</td><td>String</td><td>Specifies the codec level:<table><thead><tr><th>Codec</th><th>Level</th></tr></thead><tbody><tr><td>\"h.265/h.265\"</td><td>4.1, 4.2, 5.0, 5.1</td></tr><tr><td>\"vp9\"</td><td>3.0, 3.1, 4.0, 4.1, 5.0, 5.1</td></tr></tbody></table></td><td>Optional</td></tr><tr><td>ChCnt</td><td>integer</td><td>Specifies the required number of audio channels</td><td>Optional</td></tr><tr><td>PassThru</td><td>integer</td><td>Specifies whether the bitstream needs to be decoded on the device. By default, this reports the codec properties of both the Roku device and the attached HDMI device.<br><br>You can filter the values returned by including one of the following options:<ul><li><strong>0</strong> - Reports only codecs that are supported on the Roku device itself.</li><li><strong>1</strong> - Reports only codecs of the attached HDMI device (for example, an audio receiver).</li></ul></td><td>Optional</td></tr><tr><td>Atmos</td><td>Integer</td><td>Specifies whether atmos is required. Include 'atmos=1' if it is.</td><td>Optional</td></tr><tr><td>SampleRate</td><td>integer</td><td>Specifies the sample rate</td><td>Optional</td></tr><tr><td>BitRate</td><td>integer</td><td>Specifies the bit rate in Kbit/sec</td><td>Optional</td></tr><tr><td>Container</td><td>string</td><td>Specifies the container format</td><td>Optional</td></tr></tbody></table>",
```

Comment:
Should we be preventing adding these as raw html? (all the table html here)

Reply:
Added `node-html-markdown`